src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DetectPreviewFeatureTests.Properties.cs

```diff
@@ -533,5 +533,99 @@ class Derived: Program
             test.ExpectedDiagnostics.Add(VerifyCS.Diagnostic(DetectPreviewFeatureAnalyzer.OverridesPreviewMethodRule).WithLocation(1).WithArguments("AProperty", "Program.AProperty"));
             await test.RunAsync();
         }
+
+        [Fact]
+        public async Task TestRefProperty()
```

Comment:
Just noting that `ref T Property` isn't a bug case here; I initially thought it might have been however and there was no coverage for it.